lib/css-assembler.js

```diff
@@ -76,10 +83,19 @@ function assemble(cssFiles, absolutePath, type, callback) {
             Fs.readFile(file, {encoding: 'utf-8'}, function(err, content) {
                 var matches = [],
                     match,
-                    importCssFileAlias;
+                    importCssFileAlias,
+                    cssFile;
 
                 if (! err) {
-                    ret[cssFileAlias + ext] = content;
+
+                    //Ensure all import paths are Unix paths for prod compat.
+                    if (options.bundle) {
+                        cssFile = Upath.toUnix(cssFileAlias + ext);
+                    } else {
+                        cssFile = cssFileAlias + ext;
```

Comment:
I see. I did a quick search for where this method was getting called but I only found one for some reason, my bad.